apps/console/src/features/applications/components/settings/sign-on-methods/script-based-flow/template-description.tsx

```diff
@@ -167,7 +202,7 @@ export const TemplateDescription: FunctionComponent<TemplateDescriptionPropsInte
                                         .map(([ param, description ], index: number) => {
                                             return (
                                                 <Table.Row key={ index }>
-                                                    <Table.Cell>{ param }</Table.Cell>
+                                                    <Table.Cell><Code>{ param }</Code></Table.Cell>
```

Comment:
```suggestion
                                                    <Table.Cell>
                                                        <Code>{ param }</Code>
                                                    </Table.Cell>
```
Let's take the `<Code>` element to a newline.

Reply:
Resolved in [0aad4b3](https://github.com/wso2/identity-apps/pull/3777/commits/0aad4b38955f51616d9d2927bdaf9c64a2dbfe54)